curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console.english.md

```diff
@@ -44,17 +41,14 @@ tests:
 
 ```js
 // Open your browser console
-let outputTwo = "This will print to the browser console 2 times";
-// Use console.log() to print the outputTwo variable
-
-
-let outputOne = "Try to get this to log only once to the browser console";
-// Use console.clear() in the next line to print the outputOne only once
+let output = "Get this to log once in the browser console and twice in the freeCodeCamp console";
+// Use console.clear() on the next line to clear the browser console.
 
 
-// Use console.log() to print the outputOne variable
+// Use console.log() to print the output variable
 
 
+// Check the two consoles to see the difference. The freeCodeCamp console should have printed the variable twice, one for each test of this challenge. The browser console should only print the variable once becuase you cleared it first.
```

Comment:
```suggestion
// Check the two consoles to see the difference. The freeCodeCamp console should have printed the variable twice, once for each test of this challenge. The browser console should only print the variable once because you cleared it first.
```